autogen/Cargo.toml

```diff
@@ -6,6 +6,7 @@ authors = [
 	"Lei Zhang <antiagainst@gmail.com>",
 ]
 edition = "2018"
+rust-version = "1.58"
```

Comment:
3 years later, I don't understand why I added this: `autogen` is the only crate that _is not_ published and shouldn't care about the version at all, since Rust developers contributing to this repository and specifically using the generator to regenerate its contents can easily use the latest `stable` at all times.